src/Entity/User.php

```diff
@@ -234,4 +241,14 @@ public function getApiToken(): ?string
     {
         return $this->apiToken;
     }
+
+    public function getDeletedAt(): ?\DateTime
+    {
+        return $this->deletedAt;
+    }
+
+    public function setDeletedAt(?\DateTime $deletedAt): void
```

Comment:
please make this setter fluent too (returns self), if you have all the other setters fluent